### PR TITLE
[FUSEQE-8017] - Dynamic wait until the integration was actual triggered

### PR DESCRIPTION
**Pull request by mkralik3**
* Webhook step contains this wait implicitly

//test: `@webhook`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Conditional flows - content base routing | Conditional flows - functional test 
Conditional flows - content base routing | Conditional flows - functional test 
Conditional flows - content base routing | Conditional flows - multiple flows 
Integration - Webhook to DB | Webhook to db with middle db step 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Webhook to DB | Webhook to db with middle db step 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/resources/features/integrations/api-provider.feature [hunk 1]
```diff
@@ -238,6 +238,7 @@ Feature: API Provider Integration
       | task2 |
     Then wait until integration "TODO Integration get collection" gets into "Running" state
     When execute GET on API Provider route i-todo-integration-get-collection endpoint "/api/"
+    And wait until integration i-todo-integration-get-collection processed at least 1 message
```
Comment (asmigala) [inline ui-tests/src/test/resources/features/integrations/api-provider.feature]: I don't think it's necessary to wait for this in the api provider tests - the above `execute GET` will trigger the integration flow, if it doesn't, there's no point in waiting for anything

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Conditional flows - content base routing | Conditional flows - functional test 
Conditional flows - content base routing | Conditional flows - multiple flows 
Integration - DB to DB | Read and update operations 
Integration - Webhook to API connector | Webhook to Custom API connector 
Integration - Webhook to DB | Webhook to db with middle db step 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
MongoDB | MongoDB insert and consume tail 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Activity | Check activity 
Metrics | Check metrics 
Metrics | Check error 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Activity | Check activity 
Activity | Check whether shows error when data mapper is before 
Metrics | Check metrics 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
MongoDB | MongoDB consume stream 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
MongoDB | MongoDB consume stream 
Activity | Check activity 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Metrics | Check metrics 
Metrics | Check error 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Activity | Check activity 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Metrics | Check metrics 
Metrics | Check error 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**